WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -158,7 +159,12 @@ private async Task<List<AliceClient>> ConfirmConnectionsAsync(IEnumerable<AliceC
 
 		private IEnumerable<Money> DecomposeAmounts(FeeRate feeRate)
 		{
-			return Coins.Select(c => c.Amount - feeRate.GetFee(c.ScriptPubKey.EstimateInputVsize()));
+			var allDenominations = BaseDenominationGenerator.Generate();
+			GreedyDecomposer greedyDecomposer = new(allDenominations);
```

Comment:
Note that this will produce outputs that are slightly lower than the nominal values in `allDenominations` with the current implementation of `GreedyDecomposer`. They will be the same for all users in a single TX, but they will not be uniform between transactions, which may limit input side privacy.